src/js/popup.js

```diff
@@ -115,21 +147,27 @@ async function showRelayPanel(tipPanelToShow) {
     upgradeButtonEl.textContent = panelStrings.upgradeButton;
     upgradeButtonIconEl.src = panelStrings.upgradeButtonIcon;
 
-    //Premium Panel
+    //Premium Panel content
     registerDomainImgEl.src = panelStrings.registerDomainImg;
     aliasesUsedValEl.textContent = aliasesUsedVal;
     emailsBlockedValEl.textContent = emailsBlockedVal;
     emailsForwardedValEl.textContent = emailsForwardedVal;
 
-    //Show premium panel state
-    if (premium && premiumEnabledString === "True"){
-      premiumPanelWrapper.classList.remove("is-hidden");
-      premiumPanelWrapper.querySelectorAll(".is-hidden").forEach(premiumFeature => 
-        premiumFeature.classList.remove("is-hidden") 
-        );
-      //Toggle register domain or education module
-      checkUserSubdomain(premiumSubdomainSet);
+
+    //If Premium features are not available, do not show upgrade CTA on the panel
+    if (!premiumFeaturesAvailable(premiumEnabledString)) {
+      const premiumCTA = document.querySelector(".premium-cta");
+      const upgradeBanner = document.querySelector(".upgrade-banner-wrapper");
+      premiumCTA.classList.add("is-hidden");
+      upgradeBanner.classList.add("is-hidden");
+    }
+
+    // Remove panel status if user has unlimited aliases, so no negative alias left count
+    if (premium) {
```

Comment:
This hides a specific UI element so long as a user has unlimited alias so there would be no negative alias left count shown.